tox/__init__.py

```diff
@@ -1,8 +1,10 @@
-#
-__version__ = '2.6.1.dev1'
+from pkg_resources import get_distribution
 
 from .hookspecs import hookspec, hookimpl  # noqa
 
+_version = get_distribution('tox').version
```

Comment:
Perhaps we should use the "file generation feature" instead (example [here](https://github.com/pytest-dev/pytest-mock/blob/master/setup.py#L17))? I really like that solution due to its simplicity and avoiding having a runtime dependency with `pkg_resources`, which is widely known to be really slow to import, see pypa/setuptools#926. It also poses problems to frozen applications (testing-cabal/mock#385), although I don't think people freezes `tox`.